From b83f9eedafcfe9dc9f14fbd590e022c89c2d2d20 Mon Sep 17 00:00:00 2001
From: fafryd <dz1125.bug.tracker@gmail.com>
Date: Sun, 20 Jan 2019 17:05:04 +0100
Subject: [PATCH] meson: fix project meson_version

The Meson build system
Version: 0.49.0
...
WARNING: Project targetting '>= 0.42.0' but tried to use feature introduced in '0.46.0': compiler.has_link_argument
WARNING: Project targetting '>= 0.42.0' but tried to use feature introduced in '0.46.0': compiler.has_multi_link_argument
...
WARNING: Project specifies a minimum meson_version '>= 0.42.0' but uses features which were added in newer versions:
 * 0.46.0: {'compiler.has_link_argument', 'compiler.has_multi_link_argument'}
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 29794f083..52f4c3c5a 100644
--- a/meson.build
+++ b/meson.build
@@ -4,7 +4,7 @@ project('xserver', 'c',
             'c_std=gnu99',
         ],
         version: '1.20.99.1',
-        meson_version: '>= 0.42.0',
+        meson_version: '>= 0.46.0',
 )
 add_project_arguments('-DHAVE_DIX_CONFIG_H', language: 'c')
 cc = meson.get_compiler('c')
-- 
2.20.1

